### ISSUE TITLE
Use wget in instructions instead of curl

### ISSUE THREAD

**Issue by beruic** [yarnpkg/website#902]
Curl is no longer installed by default on Ubuntu, and AFAIK neither curl nor wget is installed by default on Debian.
It therefore makes sense to make most peoples experience with installing easier, by writing

```sh
wget -O- -q https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
```
in instrucitons instead of

```sh
curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
```

At least for the Ubuntu/Debian instructions.

**Comment by Daniel15** [comment]
Indeed, looks like `wget` is installed on a lot more Debian systems compared to `curl`, at least according to the (opt-in) popularity contest: https://qa.debian.org/popcon-graph.php?packages=wget+curl&show_installed=on&want_legend=on&want_ticks=on&from_date=&to_date=&hlght_date=&date_fmt=%25Y-%25m&beenhere=1

![image](https://user-images.githubusercontent.com/91933/51433961-22c48300-1c0c-11e9-8a8a-a431d4f43948.png)

Would you like to submit a pull request updating the docs?
